### PR TITLE
including new entity mapping information

### PR DESCRIPTION
**Pull request by shainw**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/AuditLogs/MaliciousOAuthApp_O365AttackToolkit.yaml [hunk 1]
```diff
@@ -60,3 +60,13 @@ query: |
   | project GrantAuthentication, GrantOperation, CorrelationId
   ) on CorrelationId
   | project TimeGenerated, GrantConsentType, GrantScope1, GrantInitiatedBy, AppDisplayName, AppReplyURLs, GrantIpAddress, GrantUserAgent, AppClientId, GrantAuthentication, OperationName, GrantOperation, CorrelationId, ConsentFull
+  | extend timestamp = TimeGenerated, AccountCustomEntity = GrantInitiatedBy, IPCusomtEntity = GrantIpAddress
```
Comment (ashwin-patil) [inline Detections/AuditLogs/MaliciousOAuthApp_O365AttackToolkit.yaml:63]: Typo - IPCusomtEntity

File: Detections/AuditLogs/MaliciousOAuthApp_PwnAuth.yaml [hunk 2]
```diff
@@ -60,3 +60,13 @@ query: |
   | project GrantAuthentication, GrantOperation, CorrelationId
   ) on CorrelationId
   | project TimeGenerated, GrantConsentType, GrantScope1, GrantInitiatedBy, AppDisplayName, AppReplyURLs, GrantIpAddress, GrantUserAgent, AppClientId, GrantAuthentication, OperationName, GrantOperation, CorrelationId, ConsentFull
+  | extend timestamp = TimeGenerated, AccountCustomEntity = GrantInitiatedBy, IPCusomtEntity = GrantIpAddress
```
Comment (ashwin-patil) [inline Detections/AuditLogs/MaliciousOAuthApp_PwnAuth.yaml:63]: Typo - IPCusomtEntity

File: Detections/AuditLogs/NewAppOrServicePrincipalCredential.yaml [hunk 3]
```diff
@@ -44,3 +44,13 @@ query: |
   //| where targetType =~ "Application" // or targetType =~ "ServicePrincipal"
   | project-away keyEvents
   | project-reorder TimeGenerated, OperationName, InitiatingUserOrApp, InitiatingIpAddress, UserAgent, targetDisplayName, targetId, targetType, keyDisplayName, keyType, keyUsage, keyIdentifier, CorrelationId, TenantId
+  | extend timestamp = TimeGenerated, AccountCustomEntity = InitiatingUserOrApp, IPCusomtEntity = InitiatingIpAddress
```
Comment (ashwin-patil) [inline Detections/AuditLogs/NewAppOrServicePrincipalCredential.yaml:47]: Typo - IPCusomtEntity

File: Detections/AuditLogs/SuspiciousOAuthApp_OfflineAccess.yaml [hunk 4]
```diff
@@ -58,3 +58,13 @@ query: |
   | project GrantAuthentication, GrantOperation, CorrelationId
   ) on CorrelationId
   | project TimeGenerated, GrantConsentType, GrantScope1, GrantInitiatedBy, AppDisplayName, AppReplyURLs, GrantIpAddress, GrantUserAgent, AppClientId, GrantAuthentication, OperationName, GrantOperation, CorrelationId, ConsentFull
+  | extend timestamp = TimeGenerated, AccountCustomEntity = GrantInitiatedBy, IPCusomtEntity = GrantIpAddress
```
Comment (ashwin-patil) [inline Detections/AuditLogs/SuspiciousOAuthApp_OfflineAccess.yaml:61]: Typo - IPCusomtEntity

**Review by ashwin-patil — CHANGES_REQUESTED** [review]
Other than Typo - rest looks good.

**Review by laithhisham — CHANGES_REQUESTED** [review]
Looks great
just the typos that Ashwin mentioned

File: Detections/AuditLogs/SuspiciousOAuthApp_OfflineAccess.yaml (hunk 4 above)
Comment (shainw) [reply Detections/AuditLogs/SuspiciousOAuthApp_OfflineAccess.yaml]: thanks!

File: Detections/AuditLogs/NewAppOrServicePrincipalCredential.yaml (hunk 3 above)
Comment (shainw) [reply Detections/AuditLogs/NewAppOrServicePrincipalCredential.yaml]: thanks!

File: Detections/AuditLogs/MaliciousOAuthApp_PwnAuth.yaml (hunk 2 above)
Comment (shainw) [reply Detections/AuditLogs/MaliciousOAuthApp_PwnAuth.yaml]: thanks!

File: Detections/AuditLogs/MaliciousOAuthApp_O365AttackToolkit.yaml (hunk 1 above)
Comment (shainw) [reply Detections/AuditLogs/MaliciousOAuthApp_O365AttackToolkit.yaml]: thanks!

**Review by laithhisham — APPROVED** [review]

**Pull request merged**